ra-ui-materialui cannot resolve '@mui/utils'

<!-- Please do not submit support requests or "How to" questions here. For that, 
- go to Stack Overflow: https://stackoverflow.com/questions/tagged/react-admin),  or
  go to the react-admin discord server https://discord.gg/GeZF9sqh3N for community support or
- use the Professional Support (https://react-admin-ee.marmelab.com/#support) if you're an Enterprise Edition subscriber. -->

**What you were expecting:**
No deps missing

**What happened instead:**
Module not found: Can't resolve '@mui/utils'

**Steps to reproduce:**
"next": "^14.2.14", "react-admin": "^5.2.2", getting error `Module not found: Can't resolve '@mui/utils'`

```
../node_modules/.pnpm/ra-ui-materialui@5.2.2_iqhxljt7jkst3fowqbnyte7tki/node_modules/ra-ui-materialui/dist/esm/theme/defaultTheme.js:1:1
Module not found: Can't resolve '@mui/utils'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
../node_modules/.pnpm/ra-ui-materialui@5.2.2_iqhxljt7jkst3fowqbnyte7tki/node_modules/ra-ui-materialui/dist/esm/theme/index.js
../node_modules/.pnpm/ra-ui-materialui@5.2.2_iqhxljt7jkst3fowqbnyte7tki/node_modules/ra-ui-materialui/dist/esm/index.js
../node_modules/.pnpm/react-admin@5.2.2_@types+react@18.3.11_react-dom@18.2.0_react@18.2.0__react-is@18.3.1_react@18.2.0_supports-color@9.4.0/node_modules/react-admin/dist/esm/index.js
./src/app/ReactAdmin.tsx
```

**Environment**

* React-admin version: 5.2.2
* Last version that did not exhibit the issue (if applicable): 4.16.20
* React version: 18.2.0
* Browser: chrome
* Stack trace (in case of a JS error):

--- a/packages/ra-ui-materialui/package.json
+++ b/packages/ra-ui-materialui/package.json
@@ -28,6 +28,7 @@
     "devDependencies": {
         "@mui/icons-material": "^5.15.20",
         "@mui/material": "^5.15.20",
+        "@mui/utils": "^5.15.20",
         "@testing-library/react": "^15.0.7",
         "@types/dompurify": "^3.0.2",
         "@types/react": "^18.3.3",
@@ -54,6 +55,7 @@
     "peerDependencies": {
         "@mui/icons-material": "^5.15.20",
         "@mui/material": "^5.15.20",
+        "@mui/utils": "^5.15.20",
         "ra-core": "^5.0.0",
         "react": "^18.0.0 || ^19.0.0",
         "react-dom": "^18.0.0 || ^19.0.0",

